content/documentation/staging/features/index.adoc

```diff
@@ -392,84 +392,241 @@ Check the complete link:/documentation/staging/validations/[list of validations]
+* HTTP Retry describes the retry policy to use when a HTTP request fails.
+
+++++
+<div style="display: flex;">
+ <span style="margin: 0 auto;">
+  <a class="image-popup-fit-height" href="/images/documentation/features/wizard-request-timeouts-v1.24.0.png" title="Request Timeouts">
+   <img src="/images/documentation/features/wizard-request-timeouts-v1.24.0.png" style="width: 1333px;display:inline;margin: 0 auto;" />
+  </a>
+ </span>
+</div>
+++++
+
+==== Traffic Management: Gateways
+
+Traffic Management Wizards have an Advanced Options section that can be used to extend the scenario.
+
+One available Advanced Option is to expose a Service to external traffic through an existing Gateway or create a new Gateway for this Service.
+
+++++
+<div style="display: flex;">
+ <span style="margin: 0 auto;">
+  <a class="image-popup-fit-height" href="/images/documentation/features/wizard-advanced-options-gateways-v1.24.0.png" title="Request Timeouts">
+   <img src="/images/documentation/features/wizard-advanced-options-gateways-v1.24.0.png" style="width: 1333px;display:inline;margin: 0 auto;" />
+  </a>
+ </span>
+</div>
+++++
+
+==== Traffic Management: Circuit Breaker
+
+Traffic Management Wizards allows defining Circuit Breakers on Services as part of the available Advanced Options.
+
+* Connection Pool defines the connection limits for an upstream host.
+* Outlier Detection implements the Circuit Breaker based on the consecutive errors reported.
+
+++++
+<div style="display: flex;">
+ <span style="margin: 0 auto;">
+  <a class="image-popup-fit-height" href="/images/documentation/features/wizard-advanced-options-gateways-v1.24.0.png" title="Circuit Breakers">
+   <img src="/images/documentation/features/wizard-advanced-options-gateways-v1.24.0.png" style="width: 1333px;display:inline;margin: 0 auto;" />
+  </a>
+ </span>
+</div>
+++++
+
+==== Security: Traffic Policy
+
+Traffic Management Advanced Options allows defining Security and Load Balancing settings.
 
+* TLS related settings for connections to the upstream service.
+* Automatically generated a PeerAuthentication resource for this Service.
+* Load balancing policies to apply for a specific destination.
+
+++++
+<div style="display: flex;">
+ <span style="margin: 0 auto;">
+  <a class="image-popup-fit-height" href="/images/documentation/features/wizard-advanced-options-traffic-policy-v1.24.0.png" title="Traffic Policy">
+   <img src="/images/documentation/features/wizard-advanced-options-traffic-policy-v1.24.0.png" style="width: 1333px;display:inline;margin: 0 auto;" />
+  </a>
+ </span>
+</div>
+++++
+
+=== Worksload Actions
+
+==== Automatic Sidecar Injection
+
+A *Workload* can be individually annotated to control the Sidecar Injection.
+
+A default scenario is to indicate this at *Namespace* level but there can be cases where a *Workload* shouldn't be part of the Mesh or viceversa.
+
+Kiali allows to annotate the Deployment template and propagate this configuration into the Pods.
```

Comment:
I might be wrong, but I think that the word `allow` always needs the person/people.
`Kiali allows you to` or `Kiali allows users to` or something similar.
Perhaps @jshaughn might help here

In case that is right, there are different places where this structure is used.

Reply:
I guess the structure "Kiali allows users to ..." is valid in this contex, isn't it ? Or perhaps I'm mixing something here.